save.go

```diff
@@ -364,6 +376,13 @@ func writeFile(name, body string) error {
 	return ioutil.WriteFile(name, []byte(body), 0666)
 }
 
+func printVerbose(args ...interface{}) {
```

Comment:
s/printVerbose/vprintln/

This should follow the naming convention from package fmt.